internal/resource-request-operator/broadcaster.go

```diff
@@ -246,15 +336,25 @@ func (b *Broadcaster) readPodResources(clusterID string) corev1.ResourceList {
 	return corev1.ResourceList{}
 }
 
+func (b *Broadcaster) setThreshold(threshold uint64) {
+	b.updateThresholdPercentage = threshold
+}
+
 func (b *Broadcaster) checkThreshold(clusterID string) bool {
```

Comment:
```suggestion
func (b *Broadcaster) isAboveThreshold(clusterID string) bool {
```

The function name/result meaning is not clear to me: is it checking if the value is above or belove the threshold?